### DISCUSSION TITLE
Help me to recover my github account

### DISCUSSION THREAD

**Discussion by ClanVaults** [community/community#124168] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I am reaching out to request urgent assistance in recovering access to my GitHub account. Here are the details of my situation and the information I can provide to verify my identity:

[redacted by moderator]

Problem Description:
I have lost access to my GitHub account because my phone and computer were stolen.
I do not have access to the two-factor authentication (2FA) codes or the recovery codes that I had previously downloaded.

Identity Verification:
My GitHub profile README contains a link to my LinkedIn profile: [redacted by moderator]
My GitHub profile includes a profile picture of my face, which matches the photo on my LinkedIn.
I can confirm the email address and password associated with my account.
Additionally, I can provide any other information you need to verify my identity and prove that I am the legitimate owner of the account.
I use this account for my daily work, and losing access is significantly impacting my professional activities. I would greatly appreciate your help in restoring access as soon as possible.

I am looking forward to your response and am available to provide any additional information you may need to proceed with the recovery of my account.

Thank you very much for your attention and support.

**Comment by ebndev** [comment, marked as answer]
Hi @JoaquinMarmol2,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding! 

P.S - I have also removed personal identifying information from your post.
